cpu/samd21/cpu.c

```diff
@@ -22,6 +22,10 @@
 #include "periph_conf.h"
 #include "periph/init.h"
 
+#ifndef VDD_MILLIVOLTS
+#define VDD_MILLIVOLTS 3300
+#endif
```

Comment:
I think it would be good to have an unguarded reference `#define` in sam(d|r)21-xpro's `include/periph_conf.h` right after the `CLOCK_CORECLOCK` block with a comment describing it's impact and use.